contracts/jobs/JobsManager.sol

```diff
@@ -325,9 +324,9 @@ contract JobsManager is ManagerProxyTarget, IVerifiable, IJobsManager {
 
         // Send payment to verifier if price is greater than zero
         if (price > 0) {
-            return verifierContract.verify.value(price)(_jobId, _claimId, _segmentNumber, jobs[_jobId].transcodingOptions, _dataStorageHash, _transcodedDataHash);
+            return verifierContract.verify.value(price)(_jobId, _claimId, _segmentNumber, jobs[_jobId].transcodingOptions, _dataStorageHash, _dataHashes);
```

Comment:
What is this `value(price)` syntax? I assume it just passes the value into the receiving function?

Reply:
This forwards funds to the verifier contract to be used for verification. `verify` will check that the caller provides enough funds to forward